mpi-proxy-split/lower-half/long-static_libs.txt

```diff
@@ -0,0 +1 @@
+ -lmpi_gnu_91 -lpmi /opt/cray/libfabric/1.15.0.0/lib64/libfabric.a -lcxi -ldl -latomic -lrt -lpthread /opt/cray/pe/pals/1.1.8/lib/libpals.a /global/homes/z/zz217/sw/opt/spack/cray-sles15-zen3/curl-7.80.0-vxd53vd/lib/libcurl.a /global/homes/z/zz217/sw/opt/spack/cray-sles15-zen3/readline-8.1-ib3nkkj/lib/libreadline.a /global/homes/z/zz217/sw/opt/spack/cray-sles15-zen3/readline-8.1-ib3nkkj/lib/libhistory.a /global/homes/z/zz217/sw/opt/spack/cray-sles15-zen3/zlib-1.2.11-nh5csvy/lib/libz.a /global/homes/z/zz217/sw/opt/spack/cray-sles15-zen3/openssl-1.1.1m-5yvmvee/lib/libssl.a /global/homes/z/zz217/sw/opt/spack/cray-sles15-zen3/openssl-1.1.1m-5yvmvee/lib/libcrypto.a /global/homes/z/zz217/sw/opt/spack/cray-sles15-zen3/bzip2-1.0.8-4p4fezv/lib/libbz2.a /global/homes/z/zz217/sw/opt/spack/cray-sles15-zen3/json-c-0.15-avwb2j6/lib64/libjson-c.a /global/homes/z/zz217/sw/opt/spack/cray-sles15-zen3/libzip-1.2.0-rolalf6/lib/libzip.a /global/homes/z/zz217/sw/opt/spack/cray-sles15-zen3/gdbm-1.19-v5pbqq3/lib/libgdbm_compat.a /global/homes/z/zz217/sw/opt/spack/cray-sles15-zen3/gdbm-1.19-v5pbqq3/lib/libgdbm.a -Wl,--end-group
```

Comment:
Ren=move `-Wl,--end-group` here and it back into any Makefile that uses `long-static-libs.txt`.

Reply:
@dahongli , Could you also remove `-Wl,--end-grou` from this long-static-libs.txt file?  The principle should be that we put `begin--grup` and `end-group` _only_ in Makefile's and not in files of type static-libs.txt.  Thanks.